qubesmanager/settings.py

```diff
@@ -745,6 +745,20 @@ def __init_advanced_tab__(self):
         else:
             self.dvm_template_checkbox.setVisible(False)
 
+        self.provides_network_checkbox.setChecked(
+            getattr(self.vm, 'provides_network', False))
+        if self.provides_network_checkbox.isChecked():
+            domains_using = []
+            for domain in self.vm.app.domains:
```

Comment:
There is `vm.connected_vms` you can iterate on, or simply get as a list (call `list()` on it, it is a generator).

Reply:
pushed a fix (also, I express strong disapproval about the name; it is not clear if it refers to all dependent VMs or just those that use the VM as NetVM)